arm/Microsoft.Batch/batchAccounts/readme.md

```diff
@@ -92,6 +97,7 @@ You can specify multiple user assigned identities to a resource by providing add
 | `resourceGroupName` | string | The resource group the batch account was deployed into |
 | `resourceId` | string | The resource ID of the batch account |
 
+
 ## Template references
```

Comment:
I think with the latest changes to the readme generator, the template references section is not expected anymore. @MrMCake to confirm.